src/mscorlib/src/System/IO/Path.cs

```diff
@@ -102,7 +102,12 @@ public static class Path
         // Make this public sometime.
         // The max total path is 260, and the max individual component length is 255. 
         // For example, D:\<256 char file name> isn't legal, even though it's under 260 chars.
+#if !PLATFORM_UNIX
```

Comment:
If we need to keep it constant, I am afraid we would need to have it double defined, once in the runtime / pal includes and once here and have a comment on both saying that if someone modifies it on one side, it needs to be modified on the other one as well.
But if we can make it non-constant, we could add a FCALL to the runtime that would return the platform specific value.
@jkotas what do you think?